quilt/tools/core.py

```diff
@@ -61,20 +61,16 @@ def __init__(self, children):
         assert isinstance(children, dict)
         self.children = children
 
-    def preorder_tablenodes(self):
+    def preorder(self):
```

Comment:
do you mind adding a function-comment explaining this func, esp since you're changing core behavior...

Reply:
Done.